README.md

```diff
@@ -25,3 +25,7 @@ Git command explanation:
 `git commit readme.md` Will commit the README file to the repository!
 
 'git branch <branch name>' (this will create a new branch with the desiganted name)
+
+Git command explanation: git merge
+
+'git merge  origin/feature1' (Merges two branches together into a singular branch)
```

Comment:
Please use a backtick instead of a single quotation to format the code.
<img width="269" alt="Screenshot 2023-02-25 at 10 16 08 AM" src="https://user-images.githubusercontent.com/2203934/221364898-6751098d-a89b-4c55-8983-50686934bd3c.png">
should be:
```
`git merge origin/feature1`
```

Reply:
Fixed issue by using backticks instead of single quotations when formatting code after reviewer's look at code.

Reply:
Please review again. If it works, please merge the pull request.